src/main/java/ca/corefacility/bioinformatics/irida/ria/web/announcements/AnnouncementsController.java

```diff
@@ -129,19 +127,27 @@ public String getControlCentreAdminPage(final Model model) {
      *
      * @param announcementID
      *                  The ID of the announcement to be updated
+     * @param title
+     *                  The title of the updated announcement
      * @param message
      *                  The content of the updated announcement
+     * @param priority
+     *                  The priority of the updated announcement
      * @param model
      *                  The model for the view
      * @return A redirect to the announcement control center page
      */
     @RequestMapping(value = "/{announcementID}/details", method = RequestMethod.POST)
     @PreAuthorize("hasRole('ROLE_ADMIN')")
     public String submitUpdatedAnnouncement(@PathVariable long announcementID,
+                                            @RequestParam(defaultValue = "title placeholder") String title,
```

Comment:
Is this default required here anymore?

Reply:
Right, I forgot about that placeholder. I removed the deprecated constructor and placeholder. Thanks! Please see ec4627d3f9a48108c7d1ab27900b6a3a0f6cfca3.